src/software/gui/robot_diagnostics/widgets/sensor_status.cpp

```diff
@@ -2,12 +2,12 @@
 
 void setupSensorStatus(Ui::AutoGeneratedMainWidget* widget)
 {
-    updateProgressBar(widget->progressBar_battery, 0.0f);
-    updateProgressBar(widget->progressBar_capacitor, 100.0f);
-    updateProgressBar(widget->progressBar_dribbler_speed, 0.0f);
-    updateProgressBar(widget->progressBar_break_beam, 24.24f);
-    updateProgressBar(widget->progressBar_dribbler_temp, 0.0f);
-    updateProgressBar(widget->progressBar_board_temp, 0.0f);
+    widget->progressBar_battery->setupProgressBar(20, "%1 V");
+    widget->progressBar_capacitor->setupProgressBar(200, "%1 V");
+    widget->progressBar_dribbler_speed->setupProgressBar(6000, "%1 RPM");
+    widget->progressBar_break_beam->setupProgressBar(100, "%1%");
+    widget->progressBar_dribbler_temp->setupProgressBar(100, "%1 °C");
+    widget->progressBar_board_temp->setupProgressBar(100, "%1 °C");
```

Comment:
Probably worth defaulting everything to 0 so we know the values were never set instead of having these random values.

Reply:
I think these are max values

Reply:
I think zero is more obviously "not set". If I started the application without a robot connected I'd expect to see zeros, not max values (max board temps are scary).

Reply:
on launch everything is set to 0, those are max values for the progress bar